### DISCUSSION TITLE
Single cell long read data analysis with SQANTI3

### DISCUSSION THREAD

**Discussion by vasikara17** [ConesaLab/SQANTI3#288] [Q&A] [answered]
Hello, 
thank you for the very interesting tool! 
I would like to ask if it is possible to analyze single cell data with SQUANTI3. 

Best regards,
Vasiliki

**Comment by carolinamonzo** [comment, marked as answer]
Hi @vasikara17, 
SQANTI is a QC tool, it helps understand structural categories, quality of the transcriptome etc. SQANTI is not made to analyse single cell data per se, but you can use it. For example, you could remove barcodes and UMIs and look at the whole sample's transcriptome. Or also you could separate reads according to their celltypes and do independent SQANTI runs per celltype (sort of like a pseudobulk) to compare between them etc. I don't have more ideas in mind, but SQANTI is a very versatile tool and depending on your biological question, you can use it in one way or another. But it definitely will not do cell clustering, annotation etc.
Best,
Carolina.

**Reply by vasikara17** [reply]
Hello Carolina, 
thank you very much for your reply! I did the cell clustering with the classical tools but just to check the isoform information with long reads I would try to use SQUANTI to do a basic characterization of the isoform types found.
Best,
Vasiliki
